cookbooks/aws-parallelcluster-slurm/resources/slurm_dependencies/slurm_dependencies.rb

```diff
@@ -21,10 +21,28 @@
 )
 
 action :setup do
+  package_dependencies.append(lua_devel_package())
+
   package package_dependencies do
     flush_cache({ before: true }) unless platform?('ubuntu') # not supported by apt
 
     retries 3
     retry_delay 5
   end
 end
+
+action_class do
+  def lua_devel_package
+    if platform?('ubuntu')
+      lua_lib_version =  package_version("liblua*")
```

Comment:
Who defines here which version gets installed if multiple versions are available in the repository?

Reply:
Same as above. We fixed the lua version library.